src/App.js

```diff
@@ -1,13 +1,52 @@
 import React from 'react';
+import { GoodsList } from './GoodsList';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: null,
+  };
+
+  getAllGoods = async() => {
+    const goods = await getAll();
+
+    this.setState({ goods });
+  };
+
+  getFiveGoods = async() => {
+    const goods = await get5First();
+
+    this.setState({ goods });
+  };
+
+  getRedGoods = async() => {
+    const goods = await getRed();
+
+    this.setState({ goods });
+  };
```

Comment:
You could make 1 function instead of these three, and pass callback as a parameter